packages/nextjs/components/Processes/BundlingProcess/Steps/AssetSelectionStep/AutoDetectedAssets/AutoDetectedAssetItem.tsx

```diff
@@ -57,21 +55,29 @@ export const AutoDetectedAssetItem = ({ onClick, isSelected, tx, isLoading, imag
       initial={{ opacity: 0 }}
       animate={{ opacity: 1 }}
       onClick={onClick}
-      className={`${isSelected ? "bg-base-200" : ""} ${styles.assetItem}  ${isLoading ? styles.loading : ""}`}
+      className={`${isSelected ? "bg-base-200" : ""} ${styles.assetItem}  ${false ? styles.loading : ""}`}
```

Comment:
Was replacing `isLoading` with `false` intentional?

Reply:
it was a mistake, will have that fixed